components/core/src/clp/ir/LogEvent.hpp

```diff
@@ -43,6 +48,7 @@ class LogEvent {
 private:
     // Variables
     epoch_time_ms_t m_timestamp;
+    UtcOffset m_utc_offset;
```

Comment:
Sorry, we have it initialized in the constructor, right?

Reply:
We've been adding default values for all primitives so that if someone creates a new constructor but forgets or doesn't need to update a default, they don't need to. So although this is not necessary right now since we only have one constructor, it may be necessary in the future.

Reply:
I think the default constructor of `std::chrono::duration` has the default constructor to initialize to 0.

Reply:
Unfortunately, that's not the case because the author wanted init to be a no-op (i.e., no different than using an int): https://groups.google.com/a/isocpp.org/g/std-discussion/c/OcGX7Yj3meI